src/soundbase.cpp

```diff
@@ -167,20 +167,18 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ).arg ( strSystemDriverTechniqueName ) +
-                                    "</b><br><br>" + tr ( "These are all the available drivers with error messages:" ) + "<ul>";
+            QString sErrorMessage = tr ( "<b>No usable %1 audio device found.</b><br><br>" ).arg ( strSystemDriverTechniqueName );
 
             for ( int i = 0; i < lNumDevs; i++ )
             {
-                sErrorMessage += "<li><b>" + GetDeviceName ( i ) + "</b>: " + vsErrorList[i] + "</li>";
+                sErrorMessage += "<b>" + GetDeviceName ( i ) + "</b>: " + vsErrorList[i] + "<br><br>";
             }
-            sErrorMessage += "</ul>";
 
 #if defined( _WIN32 ) && !defined( WITH_JACK )
             // to be able to access the ASIO driver setup for changing, e.g., the sample rate, we
```

Comment:
Even in the comments, I'd like not to see ASIO.  If something _is_ ASIO-specific, call a method in windows/sound.cpp.

Reply:
I think that’s what @pgScorpio tries.

Reply:
> I think that’s what @pgScorpio tries.

Yes! In the new setup all sound related error messages will come from CSound. Even sound related tooltips and what's this messages can be defined in CSound.

Reply:
I wouldn’t like to change it personally since it’s a structural change. This PR should be focused on simplifying the error message